### DISCUSSION TITLE
How to resolve import errors for DGS code-generated code (Eclipse)?

### DISCUSSION THREAD

**Discussion by lawynnj** [Project-Books/books-api#103] [Q&A] [answered]
I am getting import errors for the following dependencies:  

![Screenshot from 2021-05-05 14-09-51](https://user-images.githubusercontent.com/9859770/117189227-efda3500-adab-11eb-8293-1854614fa081.png)

It seems the modules/files do not exist in in the actual code.
![Screenshot from 2021-05-05 14-18-46](https://user-images.githubusercontent.com/9859770/117189989-d5548b80-adac-11eb-8e51-ebdd33b0f706.png)

Is there documentation on how to resolve this? Do I need to configure the `classpath` env variable?

**Comment by knjk04** [comment, marked as answer]
We're using the code generation functionality in DGS. In IntelliJ, it will usually show after building the project, so try that and let us know if the issue persists
